zeebe/qa/integration-tests/pom.xml

```diff
@@ -28,6 +28,11 @@
       <artifactId>zeebe-broker</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.camunda</groupId>
```

Comment:
Don't know why it has to be added, but then if you add it it report it as an error

Reply:
What is the error reported?

Reply:
Yes, that's why I had to add it originally..

Reply:
I retried and seems to run fine now, so I removed it